calisphere/collection_views.py

```diff
@@ -7,7 +7,7 @@
 from .facet_filter_type import FacetFilterType, TypeFF, CollectionFF
 from .utils import json_loads_url
 from .item_manager import ItemManager
-from .search_form import CollectionForm, ESCollectionForm, solr_escape
+from .search_form import CollectionForm, ESCollectionForm, query_string_escape
```

Comment:
Shouldn't this actually be removed from this line and added to line 8, so line 10:

`from .search_form import CollectionForm, ESCollectionForm`

and line 8: 

`from .utils import json_loads_url, query_string_escape`

I imagine what you have here works, but only because we've imported `query_string_escape` into `search_form`.

Reply:
Oh whoops yes, thanks for catching that!